R/position-jitter.r

```diff
@@ -48,13 +52,14 @@ PositionJitter <- ggproto("PositionJitter", Position,
   setup_params = function(self, data) {
     list(
       width = self$width %||% (resolution(data$x, zero = FALSE) * 0.4),
-      height = self$height %||% (resolution(data$y, zero = FALSE) * 0.4)
+      height = self$height %||% (resolution(data$y, zero = FALSE) * 0.4),
+      seed = self$seed
     )
   },
 
   compute_layer = function(data, params, panel) {
-    trans_x <- if (params$width > 0) function(x) jitter(x, amount = params$width)
-    trans_y <- if (params$height > 0) function(x) jitter(x, amount = params$height)
+    trans_x <- if (params$width > 0) function(x) with_seed(params$seed, jitter(x, amount = params$width))
+    trans_y <- if (params$height > 0) function(x) with_seed(params$seed, jitter(x, amount = params$height))
 
     transform_position(data, trans_x, trans_y)
```

Comment:
I think the `with_seed()` should probably go here, otherwise the x and y jitter will use the same random values